app/_src/production/dp-config/dpp.md

```diff
@@ -65,8 +65,6 @@ A `Dataplane` entity must be present for each DPP. `Dataplane` entities are mana
 - Kubernetes: The control plane {% if_version lte:2.1.x %}[**automatically generates**](/docs/{{ page.version }}/explore/dpp-on-kubernetes){% endif_version %}{% if_version gte:2.2.x %}[**automatically generates**](/docs/{{ page.version }}/production/dp-config/dpp-on-kubernetes/){% endif_version %} the `Dataplane` entity. 
 - Universal: The {% if_version lte:2.1.x %}[**user defines**](/docs/{{ page.version}}/explore/dpp-on-universal){% endif_version %}{% if_version gte:2.2.x %}[**user defines**](/docs/{{ page.version}}/production/dp-config/dpp-on-universal/){% endif_version %} the `Dataplane` entity. 
  
-For more details check the [dataplane specification](/docs/{{ page.version }}/generated/resources/proxy_dataplane):
-
```

Comment:
again whole section removed

Reply:
reference is inline now no need for the link